packages/flutter/lib/src/cupertino/sheet.dart

```diff
@@ -221,10 +221,14 @@ class CupertinoSheetTransition extends StatefulWidget {
       reverseCurve: reverseCurve,
       parent: secondaryAnimation,
     );
-    final double deviceCornerRadius = MediaQuery.maybeViewPaddingOf(context)?.top ?? 0;
+
+    final double deviceCornerRadius = (MediaQuery.maybeViewPaddingOf(context)?.top ?? 0) * .9;
```

Comment:
for these numbers, can you declare a const variable with descriptive name and use it here?

Reply:
Sure